plugins/sip/lib/model/providers/pexip/kPexipUtils.php

```diff
@@ -163,6 +212,27 @@ public static function retrieveAndValidateEntryForSipCall($queryParams, $pexipCo
 			return false;
 		}
 
+		if ($dbLiveEntry->getSipSourceType() && $dbLiveEntry->getSipSourceType() != KalturaSipSourceType::PICTURE_IN_PICTURE)
+		{
+			$dualStreamEntryId = $dbLiveEntry->getSipDualStreamEntryId();
+			if (!$dualStreamEntryId)
+			{
+				$msg = 'Dual Stream Entry is not defined on entry ' . $dbLiveEntry->getId() . ' while trying source type is '. $dbLiveEntry->getSipSourceType();
+				self::sendSipEmailNotification($dbLiveEntry->getPartnerId(), $dbLiveEntry->getPuserId(), $msg, $dbLiveEntry->getId());
+				KalturaLog::warning($msg);
+				return false;
+			}
+
+			$dbDualStreamLiveEntry = entryPeer::retrieveByPK($dualStreamEntryId);
+			if(!$dbDualStreamLiveEntry)
+			{
+				$msg = "Dual Stream entry $dualStreamEntryId was not found for sip call for entry ". $dbLiveEntry->getId() . 'with source type ' . $dbLiveEntry->getSipSourceType();
```

Comment:
All SIP msg sent to user , must be listed in one file of const values same as on ApiExceptions, so we can review / update / translate